kie-integration-test-coverage/kie-maven-plugin-tests/kjar-with-instrumentation/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.drools.testcoverage</groupId>
+    <artifactId>kie-maven-plugin-tests-parent</artifactId>
+    <version>7.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>kjar-with-instrumentation</artifactId>
+  <packaging>kjar</packaging>
+
+  <name>KIE Integration :: KIE Maven Plugin Instrumented kJAR</name>
+
+  <properties>
+    <!-- required due to kie-maven-plugin instrumentation dependencies
+         on org.drools.core.phreak.ReactiveObject and others -->
+    <enforcer.skip>true</enforcer.skip>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-maven-plugin</artifactId>
```

Comment:
I am afraid this is not going to work. The plugin itself is in the same Maven reactor build, which means it is being used before it's build (this is a limitation of Maven) and thus the parsing of the POMs will fail. This will only happen for the release versions as with SNAPSHOT Maven will just download the already available one from remote repo